### PR TITLE
Add name_en field to account setting page

### PR DESCRIPTION
**Pull request by ahmadabuwardeh**
### Description
The user should be able to add an Arabic and English full name from the "Account Settings"

## JIRA tickets
https://edraak.atlassian.net/browse/EC2020-655
https://edraak.atlassian.net/browse/EC2020-656

### CODE REVIEW

File: lms/envs/common.py [hunk 1]
```diff
@@ -3456,6 +3457,7 @@ def _make_locale_paths(settings):
 ENTERPRISE_READONLY_ACCOUNT_FIELDS = [
```
Comment (Fahminajjar) [inline lms/envs/common.py]: Is this for the user? I think it's something for enterprises right?

File: openedx/core/djangoapps/api_admin/migrations/0008_auto_20201022_1449.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.18 on 2020-10-22 11:49
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```
Comment (Fahminajjar) [inline openedx/core/djangoapps/api_admin/migrations/0008_auto_20201022_1449.py]: What is this migration? I think it's not related to what you are doing right?

File: openedx/core/djangoapps/credit/migrations/0005_auto_20201022_1449.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.18 on 2020-10-22 11:49
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```
Comment (Fahminajjar) [inline openedx/core/djangoapps/credit/migrations/0005_auto_20201022_1449.py]: Same with this migration.

**Review by Fahminajjar — CHANGES_REQUESTED** [review]

**Comment by Fahminajjar** [comment]
You just have to fix tests and make migrations for the new `name_en` field.

**Review by Fahminajjar — APPROVED** [review]

**Comment by Fahminajjar** [comment]
LGTM, we'll merge it when we merge [this PR](https://github.com/Edraak/edraak-programs/pull/1537).

File: common/djangoapps/student/admin.py [hunk 4]
```diff
@@ -217,8 +219,17 @@ def has_module_permission(self, request):
...
+    name_en = forms.CharField(required=False, widget=widgets.TextInput)
+
+    def save(self, commit=True):
+        self.instance.name_en = self.cleaned_data['name_en']
+        super(NameEnForm, self).save()
+
+
 class UserProfileInline(admin.StackedInline):
     """ Inline admin interface for UserProfile model. """
+    form = NameEnForm
```
Comment (Fahminajjar) [inline common/djangoapps/student/admin.py]: This form is for the whole UserProfile no only for `name_en` field so it's better to call it `UserProfileWithNameEnForm` or something like that. What do you think?

File: common/djangoapps/student/admin.py [hunk 5]
```diff
@@ -217,8 +219,17 @@ def has_module_permission(self, request):
         return self.has_permission(request, 'has_module_permission')
 
 
+class NameEnForm(forms.ModelForm):
+    name_en = forms.CharField(required=False, widget=widgets.TextInput)
+
+    def save(self, commit=True):
```
Comment (Fahminajjar) [inline common/djangoapps/student/admin.py]: Just add the model in the class Meta:
```python
class NameEnForm(forms.ModelForm):
    name_en = forms.CharField(required=False, widget=widgets.TextInput)

    class Meta:
        model = UserProfile

     ....
     def save(self, commit=True):
     ....
```

File: common/djangoapps/student/admin.py [hunk 6]
```diff
@@ -217,8 +219,17 @@ def has_module_permission(self, request):
...
+        self.instance.name_en = self.cleaned_data['name_en']
+        super(NameEnForm, self).save()
```
Comment (Fahminajjar) [inline common/djangoapps/student/admin.py]: You are not passing `save` method args and kwargs to the super `save`. I would recommend:
```python
def save(self, *args, **kwargs):
    ...
    super(NameEnForm, self).save(*args, **kwargs)
```

**Review by Fahminajjar — CHANGES_REQUESTED** [review]

**Review by Fahminajjar — APPROVED** [review]

**Review by Salomari1987 — APPROVED** [review]

**Review by Fahminajjar — APPROVED** [review]

**Pull request merged**